Dockerfile

```diff
@@ -8,12 +8,13 @@ WORKDIR /RackHD/on-http
 RUN mkdir -p ./node_modules \
   && ln -s /RackHD/on-core ./node_modules/on-core \
   && ln -s /RackHD/on-core/node_modules/di ./node_modules/di \
-  && npm install --ignore-scripts --production \
```

Comment:
I think it might be better to have to option to override and add dev dependencies, since they contribute quite a bit to the filesize of the image IIRC.